backend/de.metas.externalsystem/src/main/java/de/metas/externalsystem/scriptedimportconversion/housekeeping/ExternalSystemScriptedImportConversionHouseKeepingTask.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * #%L
+ * de.metas.externalsystem
+ * %%
+ * Copyright (C) 2021 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.externalsystem.scriptedimportconversion.housekeeping;
+
+import ch.qos.logback.classic.Level;
+import com.google.common.collect.ImmutableList;
+import de.metas.externalsystem.ExternalSystemConfigRepo;
+import de.metas.externalsystem.ExternalSystemParentConfig;
+import de.metas.externalsystem.ExternalSystemProcesses;
+import de.metas.externalsystem.ExternalSystemType;
+import de.metas.externalsystem.scriptedimportconversion.ScriptedImportConversionCommand;
+import de.metas.logging.LogManager;
+import de.metas.process.AdProcessId;
+import de.metas.process.IADProcessDAO;
+import de.metas.process.ProcessInfo;
+import de.metas.user.UserId;
+import de.metas.util.Loggables;
+import de.metas.util.Services;
+import lombok.NonNull;
+import org.adempiere.ad.housekeeping.spi.IStartupHouseKeepingTask;
+import org.slf4j.Logger;
+import org.springframework.stereotype.Component;
+
+import static de.metas.externalsystem.process.InvokeExternalSystemProcess.PARAM_CHILD_CONFIG_ID;
+import static de.metas.externalsystem.process.InvokeExternalSystemProcess.PARAM_EXTERNAL_REQUEST;
+
+@Component
+public class ExternalSystemScriptedImportConversionHouseKeepingTask implements IStartupHouseKeepingTask
+{
+	private static final Logger logger = LogManager.getLogger(ExternalSystemScriptedImportConversionHouseKeepingTask.class);
+
+	private final IADProcessDAO adProcessDAO = Services.get(IADProcessDAO.class);
+	private final ExternalSystemConfigRepo externalSystemConfigDAO;
+
+	public ExternalSystemScriptedImportConversionHouseKeepingTask(@NonNull final ExternalSystemConfigRepo externalSystemConfigDAO)
+	{
+		this.externalSystemConfigDAO = externalSystemConfigDAO;
+	}
+
+	@Override
+	public void executeTask()
+	{
+		final AdProcessId processId = adProcessDAO.retrieveProcessIdByClassIfUnique(
+				ExternalSystemProcesses.getExternalSystemProcessClassName(ExternalSystemType.ScriptedImportConversion));
+
+		if (processId == null)
+		{
+			Loggables.withLogger(logger, Level.DEBUG).addLog("Nothing to do!");
+			return;
+
+		}
+
+		final ImmutableList<ExternalSystemParentConfig> parentConfigList = externalSystemConfigDAO.getActiveByType(ExternalSystemType.ScriptedImportConversion);
+
+		parentConfigList
```

Comment:
imho we shall capture locally the state of each endpoint (started, stopped) and we shall send that to external systems.

More, maybe we would need a different "command" than EnableRestAPI or DisableRestAPI, 
something like InitRestAPIStatus where we send a list of endpoints we expect to be enabled.
All the others we expect to be disabled.